config/config.yaml

```diff
@@ -103,29 +116,39 @@ sinker:
 
 deck:
   spyglass:
-    size_limit: 500000000 # 500MB
+    size_limit: 100000000 # 100MB
```

Comment:
Has this change been tested?

Reply:
Not yet, I've been using the git<->prow config sync to make changes to the config.
So I test by commit to this repo.